src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -70,11 +66,25 @@ public class ImportService : IImportService
                 numRows,
                 message);
 
-            pQueue.AddMessage(new CloudQueueMessage(JsonConvert.SerializeObject(message)));
+            await _queueService.AddMessageAsync("imports-pending", message);
```

Comment:
Cool, it's really good to clean this up especially since it made it really difficult to test with the CloudStorageAccount class.

Reply:
My main reason for doing it :D